react/features/base/config/interfaceConfigWhitelist.js

```diff
@@ -69,5 +69,6 @@ export default [
     'VERTICAL_FILMSTRIP',
     'VIDEO_LAYOUT_FIT',
     'VIDEO_QUALITY_LABEL_DISABLED',
-    'filmStripOnly'
+    'filmStripOnly',
+    'SHOW_CHROME_EXTENSION_BANNER'
```

Comment:
please sort alphabetically

Reply:
reverted the change as the config already existed.